dev-util/shc/shc-3.9.6.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A generic shell script compiler"
+DESCRIPTION="Shc takes a script, which is specified on the command line and \
+produces C source code. The generated source code is then compiled and linked \
+to produce a stripped binary executable. \
+The compiled binary will still be dependent on the shell specified in the \
+first line of the shell code (i.e shebang) (i.e. #!/bin/sh), thus shc does not \
+create completely independent binaries."
+HOMEPAGE="https://github.com/neurobin/shc"
+COPYRIGHT="1997-2013 Francisco Garcia
+	2015-2017 Md Jahidul Hamid"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/neurobin/shc/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="da6a2a3ff4c356a61e086c616561bf681489993cab00c426bad0cfd703a68063"
+SOURCE_FILENAME="shc-$portVersion.tar.gz"
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	shc = $portVersion
+	cmd:shc
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc
+	cmd:libtoolize
+	cmd:make
+	"
+if [ "$recipeAction" = TEST ]; then
```

Comment:
Oops, sorry, I just removed my comment suggesting the inclusion of this trick... just before your last update.
That's because I just recalled that @alaviss had noticed that this trick had stopped working.
So, since bash is already activated, it might be worth dropping the trick.

That said, if you want you can add this:
```diff
 	cmd:make
 	"
+
+TEST_REQUIRES="
+	cmd:bash
+	"
+BUILD_PREREQUIRES="$BUILD_PREREQUIRES
+	$TEST_REQUIRES
+	"
+
 BUILD()
```

Or you might prefer something else: adding `cmd:bash` directly to `BUILD_PREREQUIRES`.

I don't know which of the 3 possibilities is the best.
To summarize, we can:
* Drop any reference to `cmd:bash`. _(I think I would choose this.)_ (This works fine for now, but the recipe will need to be modified if/when we change the default shell to either require `cmd:bash` or update `TEST()` to make it use the new default shell.)
* Put `cmd:bash` directly in `BUILD_PREREQUIRES`. This does not hurt but is unneeded at the moment.
* Put `cmd:bash` in `TEST_REQUIRES` and insert the temporary hack to append it to `BUILD_PREREQUIRES`. _(This is what's in the diff in the beginning of this message.)_ But this seems to be a bit excessive, after all, given that the two above alternatives work fine and are much shorter.

Reply:
Removed cmd:bash for now, if someone sees the need to have it in I'm open to suggestions :)